src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,37 @@
+import "./Footer.css"
+import logo from "./logo.svg";
+import { BottomNavigation, BottomNavigationAction } from "@mui/material";
+import { Container, Grid, Box, Button, Typography } from "@mui/material";
+import LinkedInIcon from '@mui/icons-material/LinkedIn';
+import GitHubIcon from '@mui/icons-material/GitHub';
+import ClosedCaptionOffIcon from '@mui/icons-material/ClosedCaptionOff';
+
+
+
+
+export default function Footer() {
+    return <footer>
+        <Box sx={{position:"fixed", width: "100%", bottom: "0", backgroundColor: "yellowgreen"}}>
```

Comment:
Seems like the formatting isn't perfect.

Reply:
move to const